server/plugins/Telegram.js

```diff
@@ -15,12 +15,14 @@ function run(trigger, scope, data, config, callback) {
         } else {
             //Notification formatted in Markdown for pretty notifications
             var notificationText = `*${data.agency} - ${data.alias}*\n` + 
-                                    `Message: ${data.message}`;
+                                    `${data.message}\n` + 
+                                    `[Google Maps](https://maps.google.com/maps?q=${data.message})`;
```

Comment:
Why is this here? Looks like a custom change that shouldn't be in this PR

Reply:
I agree